schema.sql

```diff
@@ -69,7 +69,11 @@ CREATE TABLE genotypes (
        quality TEXT,
 
        -- Cycledash-derived data
-       "annotations:gene_names" TEXT,
+       -- These are from the varcode annotation worker
+       "annotations:gene_name" TEXT,
```

Comment:
From a programmatic point of view, there is no reason not to include multiple gene names; but putting my biologist hat on, I think we should only show a single gene to the users. This might be a bold step forward, but I opt for single gene/variant route.

Happy to take this discussion off-line ;)